code/go/0chain.net/smartcontract/storagesc/config.go

```diff
@@ -106,8 +106,8 @@ type Config struct {
 	// MinAllocDuration is minimum possible duration of an
 	// allocation allowed by the SC.
 	MinAllocDuration time.Duration `json:"min_alloc_duration"`
-	// MaxChallengeCompletionTime is max time to complete a challenge.
-	MaxChallengeCompletionTime time.Duration `json:"max_challenge_completion_time"`
+	// ChallengeCompletionTime is max time to complete a challenge.
+	ChallengeCompletionTime time.Duration `json:"max_challenge_completion_time"`
```

Comment:
Please keep the MaxChallengeCompletionTime as the ChallengeCompletionTime is ambiguous, could be misread as the completion time of the challenge. So please change all the places to `Max*`

Reply:
This was suggested by @Sriep here:https://github.com/0chain/0chain/issues/1639 . @Sriep Do you have to say anything about this?

Reply:
The point here is that challengeCompletiontime is the amount of time the blobber has to complete a challenge. Previous this was set by the blobber and had to be no greater than the MaxChallengeCompletion time.

Recent changes have altered the meaning of MaxChallengeCompletionTime. Rather than maximum, a challenge completion time can be set, it is now the universal challenge completion time.